### PR TITLE
Render component tutorial update

### PR DESCRIPTION
**Pull request by yaustar**
Updated tutorials with the new render component for when we flip the switch

### CODE REVIEW

**Review by willeastcott — APPROVED** [review]

**Comment by willeastcott** [comment]
Nice job!

File: content/en/tutorials/terrain-generation.md [hunk 1]
```diff
@@ -142,22 +152,13 @@ Terrain.prototype.createTerrainFromHeightMap = function (img, subdivisions) {
         bufferHeight: bufferHeight
     });
 
-    var node = new pc.GraphNode();
-    var material = this.material.resource;
-
     var mesh = pc.createMesh(this.app.graphicsDevice, vertexData.positions, {
```
Comment (mvaligursky) [inline content/en/tutorials/terrain-generation.md:155]: I'd probably just use mesh API directly here:
```
var mesh = new Mesh(this.app.graphicsDevice);
mesh.setPositions(positions);
mesh.setNormals(vertexData.normals);
mesh.setUvs(0, vertexData.uvs);
mesh.setIndices(vertexData.indices);
mesh.update();
```

**Review by mvaligursky — APPROVED** [review]
looks great, approving, added a single comment.

File: content/en/tutorials/terrain-generation.md (hunk 1 above)
Comment (yaustar) [reply content/en/tutorials/terrain-generation.md]: 👍 Done, thanks!

**Comment by yaustar** [comment]
Do not merge/publish till we flip the switch :)

**Pull request merged**